tests/tests.sql

```diff
@@ -0,0 +1,147 @@
+-------------------------------
+-- 1. Create records 
+-------------------------------
+-- create crash ID 1, road type is `alley`
+insert into db.crashes_cris (crash_id, road_type_id)
+values (1, 1);
+-- create two units: vehicle and truck
+insert into db.units_cris (crash_id, unit_nbr, unit_type_id)
+values (1, 1, 1);
+insert into db.units_cris (crash_id, unit_nbr, unit_type_id)
+values (1, 2, 4);
+-- add a primary and non-primary person to each unit
+insert into db.people_cris (crash_id, unit_nbr, prsn_nbr, is_primary)
+values (1, 1, 1, true);
+insert into db.people_cris (crash_id, unit_nbr, prsn_nbr, is_primary)
+values (1, 1, 2, false);
+insert into db.people_cris (crash_id, unit_nbr, prsn_nbr, is_primary)
+values (1, 2, 1, true);
```

Comment:
same for `people_vz`, unit_id, unit_nbr, prn_nbp are all null, and there is no primary key

Reply:
im realizing maybe this is by design? which i guess makes sense. im just used to `unit_id` and `prsn_nbr` being combined to form the primary key so its weird to see them nullable here and the possibility of them being overridden/have diff values from whats in the cris table?

Reply:
i have the same question, and I think that I'm just fuzzy on what we should expect in the `vz` tables for non-editable columns like `is_primary`. `is_primary` being in the `vz` makes sense if we know that is something that we should expect to see for columns that cannot be edited.